dds_cli/file_handler_local.py

```diff
@@ -81,13 +81,21 @@ def __init__(self, user_input, temporary_destination, project):
 
     # Static methods ############## Static methods #
     @staticmethod
-    def generate_bucket_filepath(filename="", folder=pathlib.Path("")):
+    def generate_bucket_filepath(filename="", folder=pathlib.Path(""), project=""):
         """Generates filename and new path which the file will be
         called in the bucket."""
 
         # Generate new file name
-        new_name = "".join([str(uuid.uuid4().hex[:6]), "_", filename])
-        return str(folder / pathlib.Path(new_name))
+        new_name = (
+            project
+            + "/"
+            + str(folder / pathlib.Path(str(uuid.uuid5(uuid.NAMESPACE_X500, filename))))
+        )
```

Comment:
It looks like the project is a part of the path now, but since the bucket is project specific this is not needed. Also, since the project may produce length errors I'd just skip that part. Just have 
```
        new_name = (
            str(folder / pathlib.Path(str(uuid.uuid5(uuid.NAMESPACE_X500, filename))))
        )
```
This also matches the old structure so there's less risk that other stuff is affected.